react/Viewer/NoViewer/NoViewer.jsx

```diff
@@ -5,21 +5,24 @@ import DownloadButton from './DownloadButton'
 import styles from '../styles.styl'
 import { FileDoctype } from '../../proptypes'
 
-const NoViewer = ({ file, renderFallbackExtraContent }) => (
+const NoViewer = ({ file, url, renderFallbackExtraContent }) => (
   <div className={styles['viewer-noviewer']}>
     <FileIcon type={file.class} />
     <p className={styles['viewer-filename']}>{file.name}</p>
-    {renderFallbackExtraContent(file)}
+    {renderFallbackExtraContent(file, url)}
   </div>
 )
 
 NoViewer.propTypes = {
   file: FileDoctype.isRequired,
-  renderFallbackExtraContent: PropTypes.func
+  renderFallbackExtraContent: PropTypes.func,
+  url: PropTypes.string
 }
 
 NoViewer.defaultProps = {
-  renderFallbackExtraContent: file => <DownloadButton file={file} />
+  renderFallbackExtraContent: (file, url) => (
+    <DownloadButton file={file} url={url} />
+  )
```

Comment:
that default prop is weirdo...

I would not set it. And line 12 would be

`{renderFallbackExtraContent ? renderFallbackExtraContent(file, url) : <DownloadButton file={file} url={url} />}`

that would be simplier for me

Reply:
sorry, it's not inside your PR. So that's a nit